JustSaying/AwsTools/MessageHandling/SqsPublisher.cs

```diff
@@ -26,11 +26,9 @@ public class SqsPublisher : SqsQueueByName, IMessagePublisher
             _serialisationRegister = serialisationRegister;
         }
 
-        public Task PublishAsync(Message message) => PublishAsync(message, CancellationToken.None);
-
-        public async Task PublishAsync(Message message, CancellationToken cancellationToken)
+        public async Task PublishAsync(Message message, PublishMetadata metadata, CancellationToken cancellationToken)
```

Comment:
Some overloads as extensions for _with just a token_, _with just metadata_ and _just a token_ would make this less source-breaking to consume.

Reply:
Those are present in `MessagePublisherExtensions`